db/main.py

```diff
@@ -0,0 +1,42 @@
+import init_django_orm  # noqa: F401
+
+from django.db.models import QuerySet
+
+from db.models import Actor, Genre
+
+
+def main() -> QuerySet:
+    Genre.objects.bulk_create(
+        [Genre(name="Western"),
+         Genre(name="Action"),
+         Genre(name="Dramma")]
+    )
+    Actor.objects.bulk_create(
+        [
+            Actor(first_name="George", last_name="Klooney"),
+            Actor(first_name="Kianu", last_name="Reaves"),
+            Actor(first_name="Scarlett", last_name="Keegan"),
+            Actor(first_name="Will", last_name="Smith"),
+            Actor(first_name="Jaden", last_name="Smith"),
+            Actor(first_name="Scarlett", last_name="Johansson"),
+        ]
+    )
```

Comment:
Avoid code duplication, create a list with values that represent genres or first and last names, and use unpacking in list comprehension for bulk creation